TemporaryParser/TPRBBlockErrorNode.class.st

```diff
@@ -0,0 +1,28 @@
+Class {
+	#name : #TPRBBlockErrorNode,
+	#superclass : #TPRBEnglobingErrorNode,
+	#category : #'TemporaryParser-Nodes'
+}
+
+{ #category : #'as yet unclassified' }
+TPRBBlockErrorNode class >> error: aToken withNodes: aCollection [
+	| message |

```

Comment:
This method deserves a comment explaining what it does, what it returns, and why :)